src/lib/util/anchor.ts

```diff
@@ -2,8 +2,6 @@
  * Hook up a click listener to the window that, for all anchor tags
  * that has a relative HREF, uses the history API instead.
  */
-import { stripStart } from "./url";
```

Comment:
did not seem to be used in this file anymore.

Reply:
Good catch 👍